Sources/BraintreeCore/BTCoreConstants.swift

```diff
@@ -10,6 +10,8 @@ import Foundation
     /// :nodoc: This property is exposed for internal Braintree use only. Do not use. It is not covered by Semantic Versioning and may change or be removed at any time.
     public static let callbackURLScheme: String = "sdk.ios.braintree"
 
+    public static let venmoScheme: String = "com.venmo.touch.v2"
```

Comment:
small nit (take it or leave it): 
```suggestion
    public static let venmoURLScheme: String = "com.venmo.touch.v2"
```

Reply:
➕ 1 agree that adding URL would make this more clear